app/views/courses/CoursesView.coffee

```diff
@@ -71,6 +72,16 @@ module.exports = class CoursesView extends RootView
     @originalLevelMap = {}
     @urls = require('core/urls')
 
+    @hasActiveTournaments = false
+    tournaments = new CocoCollection([], { url: "/db/tournaments?memberId=#{me.id}", model: Tournament})
+    @listenToOnce tournaments, 'sync', =>
+      tournamentsByClass = (t.toJSON() for t in tournaments.models)[0]
```

Comment:
I don't quite follow this line. Is it possible for there to be an error here, or is there always a 0 index?

Reply:
it at least shapes as `[{}]` so it always contains 0 index :joy: and all lodash functions won't cause error with empty object/array

Reply:
Looks good to me!